app/src/main/java/org/mozilla/focus/session/IntentProcessor.kt

```diff
@@ -98,18 +98,18 @@ class IntentProcessor(
                     return null
                 }
 
-                return if (!UrlUtils.isUrl(dataString)) {
+                return if (!UrlUtils.isUrl(dataString!!)) {
```

Comment:
Note that this now throws/crashes if there's an ACTION_SEND intent without EXTRA_TEXT provided. Not sure if this will actually happen in the real world though.